main_server/local_server/install_wizard/sql_files/InitialMeetingsData.sql

```diff
@@ -1,21 +1,23 @@
 INSERT INTO `%%PREFIX%%_comdef_meetings_data` (`meetingid_bigint`, `key`, `field_prompt`, `lang_enum`, `visibility`, `data_string`, `data_bigint`, `data_double`) VALUES
-(0, 'meeting_name', 'Meeting Name', 'en', NULL, 'Meeting Name', NULL, NULL),
-(0, 'location_text', 'Location Name', 'en', NULL, 'Location Name', NULL, NULL),
-(0, 'location_info', 'Additional Location Information', 'en', NULL, 'Additional Location Information', NULL, NULL),
-(0, 'location_street', 'Street Address', 'en', NULL, 'Street Address', NULL, NULL),
-(0, 'location_city_subsection', 'Borough', 'en', NULL, 'Borough', NULL, NULL),
-(0, 'location_neighborhood', 'Neighborhood', 'en', NULL, 'Neighborhood', NULL, NULL),
-(0, 'location_municipality', 'Town', 'en', NULL, 'Town', NULL, NULL),
-(0, 'location_sub_province', 'County', 'en', NULL, 'County', NULL, NULL),
-(0, 'location_province', 'State', 'en', NULL, 'State', NULL, NULL),
-(0, 'location_postal_code_1', 'Zip Code', 'en', NULL, NULL, 0, NULL),
-(0, 'location_nation', 'Nation', 'en', NULL, 'Nation', NULL, NULL),
-(0, 'comments', 'Comments', 'en', NULL, 'Comments', NULL, NULL),
-(0, 'train_lines', 'Train Lines', 'en', NULL, NULL, NULL, NULL),
-(0, 'bus_lines', 'Bus Lines', 'en', NULL, NULL, NULL, NULL),
+(0, 'meeting_name', 'Meeting Name', 'en', 0, 'Meeting Name', NULL, NULL),
```

Comment:
Being explicit about `visibility` being `0` seemed better, want to make sure a fresh install still works right.